docs/proposals/configuration_changes.md

```diff
@@ -0,0 +1,71 @@
+## Configuration Changes
+
+### Goal
+Create a package for creating and managing configuration values as a value store. This will be used to remove the necessity of each package to be loaded just for configuration values. The other benefit is that this package could be a more robust solution to the configuration, Allowing us to do a better job of watching for updates to the configmap and managing subscriptions to those changes. This value store should also be usable by other parts of the application and therefore will require less usage of hard coded configuration structures and will allow the addition or removal of configuration values to not create such an issue. 
+
+### Work Items
+* Create a specific package for managing configuration 
```

Comment:
I'm assuming the configuration file stays the same, since you didn't specify any changes in the document.

Reply:
Yes, that was my hope. I was hoping that we could keep yaml because I think it looks nice and fits with the config map stuff that is already in place.

Reply:
Like I said below, I think you're right, the current yaml would look nicer than my proposed yaml + properties file mixture.